README.md

```diff
@@ -1 +1,62 @@
-# Steampipe-docs
\ No newline at end of file
+# Steampipe Docs
+
+The open-source version of the Steampipe Documentation. You can give us feedback and request for changes by submitting issues and pull requests in this repository.
```

Comment:
I think this should be a complete sentence, and link to the where the docs are.  Perhaps something like: `The [Steampipe Documentation](https://steampipe.io/docs) is open source!  You can give us feedback and request for changes by submitting issues and pull requests in this repository.`